samples/neo-pixels-x8/README.md

```diff
@@ -0,0 +1,31 @@
+# Drive Neo pixel strip (x8) from Raspberry Pi
+
+This [sample](Program.cs) demonstrates how to use the [Bitmap library](../../src/Iot.Device.Bindings/Utils/BitmapImage.cs) and the [Neo pixel binding](../../src/devices/Ws2812B/Ws2812B.cs) to drive an 8 Neo Pixel stick from a Raspberry Pi.
+
+## Run the sample
+
+This sample can be built and run with .NET Core 3.0. Use the following commands from the root of the repo:
```

Comment:
This should be elsewhere in the docs since it is not related to this device (fine to add a link or contribute to the generic docs: https://github.com/dotnet/iot/tree/master/Documentation)